src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -565,6 +580,12 @@ export const ShippingRoute: FC<ShippingProps> = props => {
 
     setShippingQuotes(null)
     setShippingQuoteId(undefined)
+
+    // If the address has already been verified and the user is editing the form,
+    // consider this a user-verified address (perform verification only once).
+    if (addressVerifiedBy) {
+      setAddressVerifiedBy(AddressVerifiedBy.USER)
+    }
```

Comment:
Good catch!

Unrelated to this PR: I feel the shipping step has become so complicated to map out what happens at what point. We should probably follow up and refactor/simplify things.

Reply:
oh, maybe i should add a test for this. Hoping we can do this in a followup on tests for the shipping step, but i noticed that still uses enzyme and i'm not sure of the current opinion of investing more there.